Source/Resource.swift

```diff
@@ -25,16 +25,25 @@
 //  Created by Lukas Schmidt on 21.07.16.
 //
 
-import Foundation
-
 /**
  `Resource` describes a remote resource of generic type.
  The type can be fetched via HTTP(s) and parsed into the coresponding model object.
  */
 public struct Resource<Model>: ResourceModeling {
+    /**
+     The request to get the remote data payload
+     */
     public let request: URLRequestConvertible
+    /**
+     Parses data into given Model
+     */
     public let parse: (_ data: Data) throws -> Model
     
+    /// Creates a type safe resource, which can be used to fetch it with `NetworkServiceProviding`
```

Comment:
Would go for the common notation with /** ... */ here

Reply:
This is the syntax which is autogenerated by Xcode